### ISSUE TITLE
Index raw files

### ISSUE THREAD

**Issue by wpt-issue-mover** [w3c/web-platform-tests#5481]
Originally posted as https://github.com/w3c/csswg-test/issues/1073 by @gsnedders on 05 Apr 2016, 22:48 UTC:

> This is needed if we're to ever create a manifest people can actually use to run the whole testsuite.

**Comment by foolip** [comment]
The manifest does include all files, closing this.

**Comment by gsnedders** [comment]
Hmm, we should probably rephrase this to mean the indices produced by the CSS build system (e.g., `css/CSS2/other-formats/xml/background-19.xml` doesn't appear in the CSS2 testsuite). We should probably have some meta-issue listing all the ways the CSS build system output differs from running the tests directly so we have something to point people at when they run into the various limitations. Thoughts?
